skrypt.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+
+IP_ADDR=`/sbin/ifconfig eth0 | grep 'inet addr' | cut -d: -f2 | awk '{print $1}'`
+
+echo "Twoj publiczny adres ip to: "
+curl http://169.254.169.254/latest/meta-data/public-ipv4 
+echo ""
+rails s -b $IP_ADDR -p 8082
```

Comment:
```suggestion
rails s -b $IP_ADDR -p 8082

```
Add new line at the end. Github doesn't like files without empty line at the end :p